packages/core/core/src/Config.js

```diff
@@ -37,16 +37,11 @@ export default class Config {
   reporters: Pipeline;
   pluginCache: Map<PackageName, any>;
 
-  constructor(config: ParcelConfig, filePath: FilePath) {
-    this.configPath = filePath;
-    this.resolvers = config.resolvers || [];
-    this.transforms = config.transforms || {};
-    this.runtimes = config.runtimes || {};
-    this.bundler = config.bundler || '';
-    this.namers = config.namers || [];
-    this.packagers = config.packagers || {};
-    this.optimizers = config.optimizers || {};
-    this.reporters = config.reporters || [];
+  constructor(config: ParcelConfig) {
```

Comment:
Deserialization broke things because the filePath was not getting passed into the constructor by the serializer